configs/_base_/datasets/kitti-mono3d.py

```diff
@@ -80,8 +80,7 @@
 val_evaluator = dict(
     type='KittiMetric',
     ann_file=data_root + 'kitti_infos_val.pkl',
-    metric='bbox',
-    pred_box_type_3d='Camera')
+    metric='bbox')
```

Comment:
Why do we need change pred_box_type_3d?

Reply:
I have removed the `pred_box_type_3d` argument in `KittiMetric` as it is a redundant parameter.